README.md

```diff
@@ -111,40 +111,39 @@ Table of Contents
 
 
 
-Kumari & Hoffman          Expires July 27, 2019                 [Page 2]
+Kumari & Hoffman          Expires July 28, 2019                 [Page 2]
 
-Internet-Draft            Running Root on Local             January 2019
+Internet-Draft              Root Server Local               January 2019
 
 
    root server to get the information for that TLD, or to find out that
    the TLD does not exist.  Research shows that the vast majority of
    queries going to the root are for names that do not exist in the root
-   zone, partially because the negative answers are cached for a much
-   shorter period of time.  A slow path between the recursive resolver
-   and the closest root server has a negative effect on the resolver's
-   customers.
+   zone, partially because the negative answers are sometimes cached for
```

Comment:
The "partially" leaves me wondering "well then, what are the other reasons?" - perhaps we should just remove this sentence?
Don't feel strongly....

Reply:
Agree. Fixed in the new version.